packages/components/psammead-story-promo-list/src/index.jsx

```diff
@@ -0,0 +1,32 @@
+import styled from 'styled-components';
+import { C_LUNAR } from '@bbc/psammead-styles/colours';
+import {
+  GEL_SPACING,
+  GEL_SPACING_DBL,
+  GEL_SPACING_TRPL,
+} from '@bbc/gel-foundations/spacings';
+import {
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+  GEL_GROUP_4_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+
+export const StoryPromoLi = styled.li`
```

Comment:
Just realised we should do the spec
>Role list and role listitem will need to be applied to the corresponding list items due to a VoiceOver bug to reinstate the list semantics

As this will be the case for every use of this component, we should enforce it here. To do this, rather than exporting a styled component, we should export a react component.

This can be done like

```
const StyledUnorderedList = styled.ul`
  list-style-type: none;
  margin: 0;
  padding: 0;
`;

export const StoryPromoUl = ({ children, ...props }) => {
  <StyledUnorderedList role="list" {...props} />
    {children}
  </StyledUnorderedList>
}
```

Reply:
May I ask why do we need `...prop` here?

Reply:
Just to allow the user to add any extra props if they wanted, gives more freedom really